### ISSUE TITLE
E8b - Current day indicated with colour

### ISSUE THREAD

**Issue by jpinsent** [brharp/hjckrrh#187]
On the calendar, the current day is indicated with an emboldened brown outline. This is not indicated by the screen reader in any way, failing WCAG 2.0 criterion 1.1.1 Non-text Content. 

There is no textual description of this outline present on the page either, but sighted users are able to infer that the outline indicates the current day, whereas screen reader users will have no idea it is even present. There needs to be a text alternative associated with this outline or it should be removed.

![today](https://cloud.githubusercontent.com/assets/12450480/9588305/768e778e-4ff5-11e5-8aa0-752a642e28f6.PNG)
